cleanup-passes/package-json.ts

```diff
@@ -12,146 +12,67 @@
-  };
-
-  const readJson = (filePath: string) => {
-    const fullPath = path.join(element.dir, filePath);
-    if (fs.existsSync(fullPath)) {
-      return JSON.parse(fs.readFileSync(fullPath, 'utf8'));
-    }
-    return {};
-  };
-
-  const bowerConfig: any = readJson('bower.json');
-  const npmConfig: any = readJson('package.json');
-
-  if (!bowerConfig) {
-    throw new Error('bower.json not found');
+  const npmConfigPath = path.join(element.dir, 'package.json');
+  if (await fse.pathExists(npmConfigPath)) {
+    console.log(`${element.ghRepo.name} already has a package.json, skipping.`);
+    return;
   }
 
-  // copies a property from bower.json to package.json
-  // bower.json is the source of truth
-  const copyProperty = (name: string, bowerName?: string) => npmConfig[name] =
-      bowerConfig[bowerName || name] || npmConfig[name];
-
-  npmConfig['name'] = `@polymer/${bowerConfig['name']}`;
-  npmConfig['version'] =
-      getPackageVersion(bowerConfig['name'], bowerConfig['version']);
-
-  // properties that can be directly copied
-  ['description', 'repository', 'keywords'].forEach((s) => copyProperty(s));
-
-  // authors => contributors
-  // https://docs.npmjs.com/files/package.json#people-fields-author-contributors
-  copyProperty('contributors', 'authors');
-
-  // make it public
-  delete npmConfig['private'];
-
-  // https://docs.npmjs.com/files/package.json#license
-  npmConfig['license'] = 'BSD-3-Clause';
-
-  const npmDependencies = npmConfig['dependencies'] =
-      npmConfig['dependencies'] || {};
-  const bowerDependencies = bowerConfig['dependencies'] || {};
-
-  const npmDevDependencies = npmConfig['devDependencies'] =
-      npmConfig['devDependencies'] || {};
-  const bowerDevDependencies = bowerConfig['devDependencies'] || {};
-
-  for (const bowerDep in bowerDependencies) {
-    const bowerVersion = bowerDependencies[bowerDep];
-    const npmDep = getNpmName(bowerDep, bowerVersion);
-    npmDependencies[npmDep] = getDependencyVersion(bowerDep, bowerVersion);
+  const bowerConfigPath = path.join(element.dir, 'bower.json');
+  if (!await fse.pathExists(bowerConfigPath)) {
+    throw new Error(`${element.ghRepo.name} has no bower.json.`);
   }
+  const bowerConfig: BowerConfig = await fse.readJson(bowerConfigPath);
+
+  const npmConfig: NpmConfig = {
+    // This is the style of name we're using for the 3.0 elements on NPM. Might
+    // as well be consistent, even though this is a 2.0 package.json.
+    name: `@polymer/${bowerConfig.name}`,
+    //
+    // Make sure we don't accidentally publish this repo.
+    private: true,
+
+    // Note that we exclude version because the bower version might not be well
+    // maintained, plus it won't get updated here going forward if we do more
+    // releases.
+
+    // npm warns if any of these fields aren't set.
+    description: bowerConfig.description,
+    repository: bowerConfig.repository,
+    license: bowerConfig.license,
```

Comment:
The bower config license field is not a standard SPDX license identifier that package.json should have. I'd just hard code this as "BSD-3-Clause"

Reply:
Done.